src/ds/json/adv-jsonpath.md

```diff
@@ -0,0 +1,191 @@
+[JSONPath](https://goessner.net/articles/JsonPath/) expressions help you access specific elements within a JSON document, which is similar to how XPATH works for XML documents.
```

Comment:
It is important to note that Goessner's JSONPath is not a standard. Redis JSONPath is an extension of Goessner's.

Reply:
This is made very clear in the documentation. I don't think there's a need to do so in this tutorial.